Dockerfile

```diff
@@ -108,6 +108,7 @@ RUN if grep -i -q alpine /etc/issue; then \
       apk add --no-cache ca-certificates bash curl tzdata musl-utils && \
       apk info -vv | sort; \
     elif grep -i -q ubuntu /etc/issue; then \
+      sed -i "s|deb http://.*/ \(.*\)|\0\ndeb http://mirrors.mit.edu/ubuntu/ \1\ndeb http://mirrors.ocf.berkeley.edu/ubuntu/ \1|g" /etc/apt/sources.list && \
```

Comment:
This means that the primary Ubuntu server is never used, right?

Reply:
I believe that ubuntu will prioritize the first mirror in the list, so whatever is there by default should be used first, and that is left in place because of the "\0" portion of the sed command.

Reply:
Ahhh, didn't see the `\0` :D